src/baguetteBox.js

```diff
@@ -156,9 +156,9 @@
 
     function bindImageClickListeners(selector, userOptions) {
         // For each gallery bind a click event to every image inside it
-        var gallery = document.querySelectorAll(selector);
-        galleries.push(gallery);
-        [].forEach.call(gallery, function(galleryElement) {
+        var nodeList = document.querySelectorAll(selector);
```

Comment:
I guess that's better than plain `nodeList`.

Please remove your last patch from this branch.